firmwares/aeotec/ZWA024-A.json

```
@@ -0,0 +1,31 @@
+{
+	"devices": [
+		{
+			"brand": "aeotec",
+			"model": "ZWA024-A",
+			"manufacturerId": "0x0371",
+			"productType": "0x0102",
+			"productId": "0x0018",
+			"firmwareVersion": {
+				"min": "0.0",
+				"max": "1.5"
+			}
+		}
+	],
+	"upgrades": [
+		{
+			"&if": "firmwareVersion >= 1.0 && firmwareVersion <1.6 && productId === 0x0018",

```

```suggestion
			"$if": "firmwareVersion >= 1.0 && firmwareVersion < 1.6",
```
The product ID is enforced above through `"productId": "0x0018",` already. No need to repeat this here.

---

I'll use productType instead which will help separate multiple firmware update files.